apps/spruce/src/pages/waterfall/constants.ts

```diff
@@ -1 +1,38 @@
+import { WalkthroughStep } from "components/WalkthroughGuideCue";
+
 export const VERSION_LIMIT = 5;
+
+export const waterfallGuideId = "data-waterfall-guide-id";
+
+export const walkthroughSteps: WalkthroughStep[] = [
+  {
+    title: "New Layout",
+    description:
+      "We've changed the Project Health page to a new layout for increased information density. Columns are commits, rows are build variants, and tasks are box icons.",
```

Comment:
Also, I think it makes sense to not imply we're changing the project health page. This is a new page, no need to mention it.
```suggestion
      "We've introduced a new layout for increased information density. Columns are commits, rows are build variants, and tasks are box icons.",
```